backend/glp/views.py

```
@@ -0,0 +1,310 @@
+            runs = (
+                GLPRun.objects.filter(status=GLPRun.STATUS_COMPLETED)
+                .select_related('season')
+                .order_by('-sequence')
+            )
+            payload = {'results': [run_payload(run) for run in runs]}
+            cache.set(cache_key, payload, CACHE_TTL_SECONDS)
+        return Response(payload)
+
+
+class GLPRunDetailView(GLPPublicView):
+    def get(self, request, run_id):
```

Ruff (0.16.1)

[warning] Unused method argument: `request`

(ARG002)